frontend/src/flashcard/flashcardView.js

```diff
@@ -0,0 +1,375 @@
+import React, { Component } from 'react';
+import * as PropTypes from 'prop-types';
+
+import { Footer, Navbar, LoadingPage } from '../UILibrary/components';
+
+const capitalize = (word) => {
+    return word.charAt(0).toUpperCase() + word.slice(1);
+};
+
+const filledStar = (color) => {
```

Comment:
My preference for SVGs that are used without any direct transformation in the code is to serve them as separate files, rather than inlining them in code. There are pros and cons to both approaches: inlining reduces the number of requests the browser has to make while rendering the page, but it means that the browser can't cache the assets separately from the JavaScript.

Serving these as files would mean creating `.svg` files with the just the content of the SVG tag, putting them in the static/img folder, and then serving them using an `img` tag.